packages/compat/tests/resolver.test.ts

```diff
@@ -681,14 +681,35 @@ describe('compat-resolver', function () {
       },
     ]);
   });
+  test('helper currying using the "helper" keyword', function () {
+    let findDependencies = configure({ staticHelpers: true });
+    givenFile('helpers/hello-world.js');
+    expect(
+      findDependencies(
+        'templates/application.hbs',
+        `
+        {{#let (helper "hello-world" name="World") as |hello|}}
```

Comment:
This is based on the code snippet in the [RFC](https://github.com/emberjs/rfcs/blob/master/text/0432-contextual-helpers.md).